controllers/account.controller.js

```diff
@@ -85,15 +85,15 @@ async function addUser(req, res) {
 /**
  * @async
  * @function updateAccount
- * @param {*} req
+ * @param {{params: {id: string}, body: {Object}}} req
  * @param {*} res
  * @return {JSON} Success or error status
  * @description 
  *      Change a user's account information based on the account's mongoID. 
  *      The new account information is located in req.body
  */
 async function updateAccount(req, res) {
-    const id = req.body._id;
+    const id = req.params.id;
```

Comment:
Are you sure this is params.id?

Reply:
Going to put in validation for params._id and move all to body._id in.